### PR TITLE
Code snippets for data frames library

### PR DESCRIPTION
**Pull request by sylwiabr**

### CODE REVIEW

File: snippets.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
+Dataframes.Table:
+  $global:
+    - name: read "path/to/data"
```
Comment (iamrecursion) [inline snippets.yaml]: As we already discussed for stdlib snippets, this should be made into an actual readable path when we can.

File: snippets.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      documentation: Read table from file.
+  
+  Table:
+    - name: write "./myTable.csv"
+      documentation: Save the table as a file. Supported file formats are: csv, txt, xlsx and feather.
+    - name: describeNa
+      documentation: Calculate number and percentage of null values for each column.
+    - name: describe
+      documentation: Calculate statistics: standard deviation, minimum, maximum, mean, median qunatile 0.25 and 0.75 for each column.
```
Comment (iamrecursion) [inline snippets.yaml]:
```suggestion
      documentation: Calculate statistics: standard deviation, minimum, maximum, mean, median quantile 0.25 and 0.75 for each column. 
```

File: snippets.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      documentation: Filter columns in the table.
+    - name: dropNa
+      documentation: Remove rows with null values.
+    - name: filterColumns .isNumeric . fillNA 0
+      documentation: Fill nulls with given value.
+    - name: filterColumns .isNumeric . correlations
+      documentation: Compute pairwise Pearson's correlation of columns.
+  Histogram:
+    - name: setBins 5
+      documentaion: Change number of bins for histogram. Default number of bins is 10.
```
Comment (iamrecursion) [inline snippets.yaml]:
```suggestion
      documentaion: Change the number of bins for histogram. Default: 10.
```

File: snippets.yaml (hunk 1 above)
Comment (kustosz) [reply snippets.yaml]: Also, it should be `Table.read` if it's a global snippet.

Comment (sylwiabr) [reply snippets.yaml]: https://github.com/luna/dataframes/issues/142

File: snippets.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    - name: take 1
+      documentation: Take first `n` rows.
+    - name: drop 1
+      documentation: Drop first `n` rows.
+    - name: filterColumns .isNumeric
+      documentation: Filter columns in the table.
+    - name: dropNa
+      documentation: Remove rows with null values.
+    - name: filterColumns .isNumeric . fillNA 0
+      documentation: Fill nulls with given value.
```
Comment (mwu-tow) [inline snippets.yaml]: We should unify wording "nulls" vs "null values".

File: snippets.yaml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+    - name: Table.read "path/to/data"
+      documentation: Read table from file.
+  
+  Table:
+    - name: write "./myTable.csv"
+      documentation: Save the table as a file. Supported file formats are: csv, txt, xlsx and feather.
+    - name: describeNa
+      documentation: Calculate number and percentage of null values for each column.
+    - name: describe
+      documentation: Calculate statistics: standard deviation, minimum, maximum, mean, median quantile 0.25 and 0.75 for each column.
```
Comment (mwu-tow) [inline snippets.yaml]: Not requesting a change, just wondering.  Perhaps we should omit listing the statistics? 

What is the preferred length for the description? This one feels a little long and I'm not sure to what degree we want to duplicate documentation.
Also, I'm concerned about the number of places in code that will become implicitly obsolote once we add/remove a calculated statistic.

Possible counterargument: searchabality.

**Pull request merged**